lib/main_common.pm

```diff
@@ -1309,6 +1309,7 @@ sub load_x11tests {
                 loadtest "x11/eog";
                 loadtest(is_sle('<15') ? "x11/rhythmbox" : "x11/gnome_music");
                 loadtest "x11/wireshark";
+                loadtest "x11/remote_desktop/screensharing_available" if is_sle("=15-sp4");
```

Comment:
Why ? If difference between 15sp4+ and older versions is handled in the test itself.